lib/routes/auth.js

```diff
@@ -44,4 +48,43 @@ router.post(
   }
 );
 
+// /api/auth/login
+router.post(
+  './login',
```

Comment:
```suggestion
  '/login',
```

The `.` is not required and not common here. See https://expressjs.com/en/guide/routing.html
Actually, this could be an issue if the `.` is processed as a regular expression.

Reply:
Fixed.